js/main.js

```diff
@@ -0,0 +1,34 @@
+$(function() {
+  var state = ['nyc', 'sf', 'sydney','la', 'austin'];
+
+  $('form').submit(changeBackground);
+  
+      function changeBackground() {
+        event.preventDefault()
+
+        
+        // var city = $(this).attr("id").val();
+          var city = $('#city-type').val();
+
+          console.log(city);
+          if (city === 'nyc' || 'new york city' || 'new york') {
+            $('body').attr("backgroundImage", "url(/images/nyc.jpg))");
+          
+          } else if (city === 'sf' || 'san francisco' || 'bay area') {
```

Comment:
this should be if (city === 'sf' || city = 'san francisco' || city = 'bay area') 🔨